data/actionsModels.js

```diff
@@ -0,0 +1,36 @@
+const knex = require('knex');
+const dbConfig = require('../knexfile');
+const db = knex(dbConfig.development);
+
+function get(id){
+    if(id){
+        return db('actions').where('id', id).then(action => { return action[0] })
+    }
+
+    return db('actions').then(actions => { return actions })
```

Comment:
You don't need this `then` chained on here; all you're doing is taking the actions you're already getting and re-sending them essentially!
```javascript
return db('actions')
```
will work just as well